tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -105,29 +105,27 @@ namespace tuplex {
         assert(stage);
 
         // reset history server
-        _historyServer.reset();
+//        _historyServer.reset();
 
         if(!stage)
             return;
 
         // history server connection should be established
         bool useWebUI = _options.USE_WEBUI();
         // register new job
-        if(useWebUI) {
+        if(useWebUI && stage->predecessors().size() == 0) {
+            _historyServer.reset();
             _historyServer = HistoryServerConnector::registerNewJob(_historyConn,
                     "local backend", stage->plan(), _options);
             if(_historyServer) {
                 logger().info("track job under " + _historyServer->trackURL());
-                _historyServer->sendStatus(JobStatus::SCHEDULED);
+                _historyServer->sendStatus(JobStatus::STARTED);
```

Comment:
why is scheduled not used anymore? Can you shortly comment on github on the reasoning?

Reply:
I didn't see the reasoning for it in the execute method. To me, it seems that when execute is called on the first stage, the job should be started, not scheduled.